clang/test/dpct/math/cuda-math-extension.cu

```diff
@@ -181,6 +181,25 @@ __global__ void kernelFuncFloat(float *deviceArrayFloat) {
   f2 = __fmaf_ru(f0, f1, f2);
   // CHECK: f2 = sycl::ext::intel::math::fmaf_rz(f0, f1, f2);
   f2 = __fmaf_rz(f0, f1, f2);
+  // CHECK: DPCT1138:{{[0-9]+}}: The flush-to-zero behavior can only be controlled by the compiler option in SYCL. Please verify the correctness of the migrated code.
```

Comment:
The API __fmaf_ieee_rd() ignores the flush to zero compiler option when handling denormalized data, while the migrated SYCL API is impacted by it. Please...